bindings/c/test/apitester/TesterApiWrapper.cpp

```diff
@@ -109,6 +159,85 @@ fdb_error_t Transaction::setOption(FDBTransactionOption option) {
 	return fdb_transaction_set_option(tx_.get(), option, reinterpret_cast<const uint8_t*>(""), 0);
 }
 
+class TesterGranuleContext {
```

Comment:
We plan replacing TesterApiWrapper by C++ API introduced in https://github.com/apple/foundationdb/pull/6489/ once it is merged. It means we should rely on a generic C++ API here. A specific context implementation should be rather part of the workload implementation.

Reply:
So this is not context-specific to this workload, this is a basic c++ implementation for all local file blob granule workloads that should ideally also replace mako's C implementation. So once #6489 is merged, we should probably just replace https://github.com/apple/foundationdb/pull/6489/files#diff-93f0f59d03627e2dce9dc83de872b7bbca5f894b0761b02671e22509df624ad4 with the implementation here.

Reply:
Ok, then we will have to sync on that after the both PRs are merged